modules/services/betterlockscreen.nix

```
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.betterlockscreen;
+
+in {
+  meta.maintainers = with maintainers; [ sebtm ];
+
+  options = {
+    services.betterlockscreen = {
+      enable =
+        mkEnableOption "Configure screen-locker module to use betterlockscren";
```

`mkEnableOption` automatically formats like: `Whether or not to enable %s.`

```suggestion
        mkEnableOption "betterlockscreen, a screen-locker module";
```